pyanaconda/modules/payloads/payload/rpm_ostree/installation.py

```diff
@@ -628,10 +628,17 @@ def run(self):
             if not self._data.signature_verification_enabled:
                 args.append("--no-signature-verification")
 
-            safe_exec_with_redirect(
-                "ostree",
-                args
-            )
+            # TODO refactor things so we can log stdout to the program log or journal.
+            # For now we just use anaconda's stdout.
+            # Also TODO: Add a progress/status API to bootc and use it here
+            # https://github.com/containers/bootc/issues/522
+            proc = startProgram(args, stdout=None, stderr=subprocess.PIPE)
```

Comment:
You can pass to execWithRedirect the stderr and stdout parameters and achieve the same.

Reply:
And for printing the error in the PayloadInstallationError it should be enough to catch the exception

Reply:
Are you sure? I didn't actually try it but just reading the code for `_run_program` it sure looks to me like `err_string` (i.e. the result of stderr, which is what we want to capture) never ends up being included in a thrown exception (which is what we want to show to the user).

I am fine with *any* solution that results in stderr from this process being shown to the user, to be clear.

Reply:
it's part of the output_string that it's returned (stderr is redirected for that method by default to stdout):
https://github.com/rhinstaller/anaconda/blob/master/pyanaconda/core/util.py#L192